collector/runtime-benchmarks/fmt/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use benchlib::benchmark::run_benchmark_group;
```

Comment:
Add a comment here that it's just a smoke test for formatting and should be replaced with something better eventually?

Reply:
Sure, although I guess that this can be said about most of our current runtime benchmarks :laughing: